### ISSUE TITLE
Doesn't seem to work with Godot 4.3

### ISSUE THREAD

**Issue by jasonswearingen** [MihaMarkic/godot-tscn-source-generator#1]
Hello I am new both to godot and your sourcegen package, but it doesn't seem to be able to generate code for .tscn files.   The `InputActions` class is properly generated in my project but everything else is blank.

I am adding this to my .csproj:

```xml
	  <AdditionalFiles Include="assets/**/*.tscn" />
    <AdditionalFiles Include="src/**/*.tscn" />
```

When I add `<AdditionalFiles Include="**/*.tscn" />` it seems to be able to generate code from a single `.tscn` found in an addon (PhantomCamera) before stopping.

**Comment by MihaMarkic** [comment]
Did you try adding single .tscn files?

**Comment by jasonswearingen** [comment]
yes, same thing.   however I do have other `AdditionalFiles` includes used by other sourcegenerators, but no other `.tscn` files

**Comment by jasonswearingen** [comment]
just to venture a (uneducated) guess, I'd think there has been a change in .tscn format from when you created the antlr definitions, and the current .tscn format.     That one of the PhantomCamera scenes works probably means it wasn't saved since that version you targeted

oh, and I didn't see any related messages in my build output

**Comment by MihaMarkic** [comment]
Messages are usually under dependencies I think. But anyway, can you post a repro here? At least the .tscn file that doesn't work.

**Comment by jasonswearingen** [comment]
ok I will try making an example project, it might take a day or 2

**Comment by MihaMarkic** [comment]
In case it was problem similar to #2, you might try with freshly released beta.2 version.

**Comment by MihaMarkic** [comment]
Closed for being inactive.
